jupyter_notebooks/options_flow.ipynb

```diff
@@ -802,15 +802,19 @@
     }
    ],
    "source": [
-    "data = amberdata_client.get_options_scanner_top_trades(exchange='deribit', currency='btc', startDate='2024-06-07', endDate='2024-06-12')\n",
+    "import datetime\n",
```

Comment:
@cscbai Thanks for making the date range flexible and current, this is a good addition 👍